core/psh/Makefile

```diff
@@ -32,6 +32,7 @@ $(NAME)-install:
 		echo "LINK /bin/{psh -> $$applet}"; \
 		ln -f "$$PREFIX_ROOTFS/bin/psh" "$$PREFIX_ROOTFS/bin/$$applet"; \
 	done
+	ln -f "$$PREFIX_ROOTFS/bin/psh" "$$PREFIX_ROOTFS/bin/pshlogin"
```

Comment:
this looks bad when building, maybe add separate variable `PSH_INSTALLED_APPLETS`. 
Not sure what should happen if I don't want `auth` applet to be compiled-in (should `pshlogin` be created regardless?)
![image](https://user-images.githubusercontent.com/3939983/124949607-000fca00-e012-11eb-83df-c4d384d39966.png)

Reply:
- `PSH_LINK_APPLETS` variable added. It consists of applets for which links should be created. Feature where one psh applet registers more than one command does not support creating links for each of them, thus it is only used inside pshlogin as it needs runnability from different names (also there is no particularly good way of forcing **only** one command per applet) 
 - In this approach `pshlogin` and `auth` are integral part of `pshapp` so they cannot be excluded from compilation (they are not listed on `PSH_ALLCOMMANDS` in psh Makefile)